### PR TITLE
test/e2e/apimachinery: fix staticcheck warning

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:

fix staticcheck warning

**Which issue(s) this PR fixes**:

Ref: #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by draveness — APPROVED** [review]
/kind cleanup
/priority backlog
/lgtm

File: test/e2e/apimachinery/webhook.go [hunk 1]
```diff
@@ -1659,17 +1629,6 @@ func namedNonCompliantConfigMap(name string, f *framework.Framework) *v1.ConfigM
 	}
 }
 
-func nonDeletableConfigmap(f *framework.Framework) *v1.ConfigMap {
-	return &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: nonDeletableConfigmapName,
```
Comment (liggitt) [inline test/e2e/apimachinery/webhook.go]: nonDeletableConfigmapName is unused now, right?

Comment (hwdef) [reply test/e2e/apimachinery/webhook.go]: Yes, you are right.
Thanks for your suggestion.
I update this PR.

File: test/e2e/apimachinery/webhook.go [hunk 2]
```diff
@@ -66,7 +66,6 @@ const (
 	toBeAttachedPodName       = "to-be-attached-pod"
 	hangingPodName            = "hanging-pod"
 	disallowedConfigMapName   = "disallowed-configmap"
-	nonDeletableConfigmapName = "nondeletable-configmap"
```
Comment (liggitt) [inline test/e2e/apimachinery/webhook.go]: I expect go fmt will want the rest of the items unindented, run `hack/update-gofmt.sh`

Comment (hwdef) [reply test/e2e/apimachinery/webhook.go]: ok, I have done this.

**Pull request merged**